edumanage/models.py

```diff
@@ -222,6 +320,27 @@ def __str__(self):
         return self.url
 
 
+@python_2_unicode_compatible
+class Address_i18n(models.Model):
```

Comment:
I understand the structure is driven by what eduroam.org asked for ... but shouldn't an address also specify a Country?
Street and City does not look sufficient to me in the global context....

Reply:
One might argue this can be derived considering the RO to which the particular object is linked and the country (area) they serve. I do get your point though, yet I am focused on covering eduroam db v2 schema changes in this (quite large) change set, and this can be added later.

Reply:
Sure, no problem.  I understand this is driven by the V2 structure and we can't change it easily from here - still wanted to raise it as an usability issue.  But that's not to be solved here, so I'm marking this part as resolved.